lightstep/resource_metric_condition.go

```diff
@@ -344,6 +312,67 @@ func getThresholdSchema() map[string]*schema.Schema {
 	}
 }
 
+// expression is optional because it cannot be included in composite alerts but is otherwise required
```

Comment:
I'm not sure I follow this comment. There doesn't appear to be anything marked optional here.